src/reference/asciidoc/aggregator.adoc

```diff
@@ -262,74 +247,76 @@ public interface CorrelationStrategy {
 
 }
 ----
+====
 
-The method returns an Object which represents the correlation key used for associating the message with a message group.
-The key must satisfy the criteria used for a key in a Map with respect to the implementation of `equals()` and `hashCode()`.
+The method returns an `Object` that represents the correlation key used for associating the message with a message group.
+The key must satisfy the criteria used for a key in a `Map` with respect to the implementation of `equals()` and `hashCode()`.
 
-In general, any POJO can implement the correlation logic, and the rules for mapping a message to a method's argument (or arguments) are the same as for a `ServiceActivator` (including support for @Header annotations).
+In general, any POJO can implement the correlation logic, and the rules for mapping a message to a method's argument (or arguments) are the same as for a `ServiceActivator` (including support for `@Header` annotations).
 The method must return a value, and the value must not be `null`.
 
-Spring Integration provides an out-of-the box implementation for `CorrelationStrategy`, the `HeaderAttributeCorrelationStrategy`.
+Spring Integration provides an implementation for `CorrelationStrategy`: `HeaderAttributeCorrelationStrategy`.
 This implementation returns the value of one of the message headers (whose name is specified by a constructor argument) as the correlation key.
-By default, the correlation strategy is a `HeaderAttributeCorrelationStrategy` returning the value of the `CORRELATION_ID` header attribute.
-If you have a custom header name you would like to use for correlation, then simply configure that on an instance of `HeaderAttributeCorrelationStrategy` and provide that as a reference for the Aggregator's correlation-strategy.
+By default, the correlation strategy is a `HeaderAttributeCorrelationStrategy` that returns the value of the `CORRELATION_ID` header attribute.
+If you have a custom header name you would like to use for correlation, you can configure it on an instance of `HeaderAttributeCorrelationStrategy` and provide that as a reference for the aggregator's correlation strategy.
 
-===== LockRegistry
+===== `LockRegistry`
 
-Changes to groups are thread safe; a `LockRegistry` is used to obtain a lock for the resolved correlation id.
+Changes to groups are thread safe.
+A `LockRegistry` is used to obtain a lock for the resolved correlation ID.
 A `DefaultLockRegistry` is used by default (in-memory).
-For synchronizing updates across servers, where a shared `MessageGroupStore` is being used, a shared lock registry
-must be configured.
-See <<aggregator-config>> below for more information.
+For synchronizing updates across servers, where a shared `MessageGroupStore` is being used, you must configure a shared lock registry.
+See "`<<aggregator>>`" for more information.
```

Comment:
This is wrong a bit: the `[[aggregator]]` is a link to the chapter where we are.
The `[[aggregator-config]]` was about a general section for configuration (xml, dsl etc.).
Right now I see with a couple next lines that you broke such a structure and introduced separate sections for `xml`, `dsl` etc. I understand that it may make sense when we come back to the collapsing tabs between configuration styles, but the link here is still wrong: we definitely need to point to the proper configuration position or just mention where to look. Honestly it even much easier here since the configuration part is the next paragraph 😄